### PR TITLE
Release tool: fix a few issues

### PR DESCRIPTION
**Pull request by camdencheek**
This fixes a few issues I hit during the 5.2.2 patch release.

## Test plan

I used these changes to make the release.

### CODE REVIEW

File: dev/release/src/release.ts [hunk 1]
```diff
@@ -726,7 +731,6 @@ cc @${release.captainGitHubUsername}
                                 ? `comby -in-place 'const minimumUpgradeableVersion = ":[1]"' 'const minimumUpgradeableVersion = "${release.version.version}"' dev/ci/internal/ci/*.go`
                                 : 'echo "Skipping minimumUpgradeableVersion bump on patch release"',
                             updateUpgradeGuides(release.previous.version, release.version.version),
-                            `comby -in-place 'git_versions=(:[1])' 'git_versions=(:[1] v${release.version.version})' cmd/migrator/generate.sh`,
```
Comment (camdencheek) [inline dev/release/src/release.ts:729]: This file no longer exists, and was replaced with a nicer way of doing schema stuff.

File: dev/release/src/release.ts [hunk 2]
```diff
@@ -775,9 +779,9 @@ cc @${release.captainGitHubUsername}
                         commitMessage: defaultPRMessage,
                         title: defaultPRMessage,
                         edits: [
-                            `sg ops update-images -pin-tag ${release.version.version} base/`,
-                            `sg ops update-images -pin-tag ${release.version.version} components/executors/`,
-                            `sg ops update-images -pin-tag ${release.version.version} components/utils/migrator`,
+                            `sg ops update-images -cr-username ${dockerUsername} -cr-password ${dockerPassword} -pin-tag ${release.version.version} base/`,
+                            `sg ops update-images -cr-username ${dockerUsername} -cr-password ${dockerPassword} -pin-tag ${release.version.version} components/executors/`,
+                            `sg ops update-images -cr-username ${dockerUsername} -cr-password ${dockerPassword} -pin-tag ${release.version.version} components/utils/migrator`,
```
Comment (camdencheek) [inline dev/release/src/release.ts:784]: I think this previously worked by setting environment variables, but that no longer works. Passing the creds in explicitly is a bit easier to trace dependencies anyways

File: dev/release/src/util.ts [hunk 3]
```diff
@@ -207,13 +207,13 @@ export async function getContainerRegistryCredential(registryHostname: string):
...
     )
-    const registryPassowrd = await readLine(
+    const registryPassword = await readLine(
         `Enter your container registry (${registryHostname} ) access token: `,
         `${cacheFolder}/cr_${registryHostname.replace('.', '_')}_password.txt`
     )
     const credential: ContainerRegistryCredential = {
         username: registryUsername,
-        password: registryPassowrd,
+        password: registryPassword,
```
Comment (camdencheek) [inline dev/release/src/util.ts:216]: just a typo

**Review by unknwon — APPROVED** [review]

**Review by DaedalusG — APPROVED** [review]
Ty Camden!

File: dev/release/src/release.ts (hunk 2 above)
Comment (keegancsmith) [reply dev/release/src/release.ts:784]: I'm sorry fixing this was part of my list of things to fix from the last release. The way I worked around it in the release was hacking the sg tool to hardcode my creds :O

**Review by BolajiOlajide — COMMENTED** [review]
Thank you for the fix, Camden!

**Pull request merged**